Vagrantfile

```diff
@@ -29,16 +74,17 @@ Vagrant.configure("2") do |config|
 		stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
 
 		stage.vm.provider :aws do |aws,override|
-			aws.keypair_name = "osx_rsa"
-			aws.ami = "ami-d2414e38"
-			aws.instance_type = "t2.micro"
-			aws.region = "eu-west-1"
-			aws.subnet_id = "subnet-2c67fe64"
-			aws.security_groups = ["sg-7b78fe07"]
-			aws.associate_public_ip = true
-
-			override.ssh.username = "ubuntu"
-			override.ssh.private_key_path = "~/.ssh/osx_rsa.pem"
+			aws.keypair_name = AWS['stage_keypair_name']
+			aws.ami = AWS['stage_ami']
+			aws.instance_type = AWS['stage_instance_type']
+			aws.region = AWS['stage_region']
+			aws.subnet_id = AWS['stage_subnet_id']
+			aws.security_groups = AWS['stage_security_groups']
+			aws.associate_public_ip = AWS['stage_associate_public_ip']
+			aws.access_key_id = AWS['stage_access_key_id']
```

Comment:
the access_key_id and the secret_access_key is not needed.  will explain in our coming AWS workshop.

Reply:
found the reason, fixed